src/reactA11yNoOnchangeRule.ts

```diff
@@ -0,0 +1,64 @@
+import * as ts from 'typescript';
+import * as Lint from 'tslint';
+
+import {ErrorTolerantWalker} from './utils/ErrorTolerantWalker';
+import {ExtendedMetadata} from './utils/ExtendedMetadata';
+import {getJsxAttributesFromJsxElement} from './utils/JsxAttribute';
+
+/**
+ * Implementation of the react-a11y-no-onchange rule.
+ */
+export class Rule extends Lint.Rules.AbstractRule {
+
+    public static metadata: ExtendedMetadata = {
+        ruleName: 'react-a11y-no-onchange',
+        type: 'functionality',
+        description: 'For accessibility of select menus, onChange event handler should not be used with the select element.',
+        options: 'string[]',
+        optionsDescription: 'Additional tag name to validate.',
```

Comment:
While you're in the area, would you mind slightly changing this to indicate that any number of tag name**s** are allowed? It would be easy to assume only one tag name is allowed from just seeing this description.